website/docs/reference/widgets/list.md

```diff
@@ -115,10 +149,21 @@ Properties allow you to customize the widget, connect it to other widgets and tr
-| **Server-side Pagination** | Enables you to implement pagination by limiting the number of results fetched per Query request.  |
+| **Data Identifier** | Like `keys` in React, you need to select a data identifier from your dataset, which helps uniquely identify each item. This helps the List widget identify which items are added, have changed, or are removed. You can also combine two columns or data identifiers by enabling the `JS` mode. |
+| **Server-side Pagination** | Enables you to implement pagination by limiting the number of results fetched per query request.  |
 | **Visible**                | Controls widget's visibility on the page. When turned off: The widget is visible when the app is published. It appears translucent when in Edit mode.                                                                                            |
 | **Animate Loading**        | When turned off, the widget loads without any skeletal animation. You can use a toggle switch to turn it on/off. You can also turn it off/on using javascript by enabling the JS label next to it.                                                    |
 
+### Internal properties
+
+These properties are available only to the widgets placed inside the List widget and enable you to configure the widget's properties based on the position/order of the item.
+
+| Property    | Description                                                                                                                                                                                                                               |
+| ------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **currentItem** | This represents the data for a particular item.                                                                                                                                                   |
+| **currentIndex**         | Represents the index of the item.                                                                                                                                                                               |
+| **currentView**       | Reflects the state of all the widgets present in the current item. This property can be used to access all sibling widgets present inside a List item card.                                                                                                                      |
+| **level_***           | This property is only available for nested lists where \* represents the level number (from 1 through 3). This property can be used to access the **currentItem**, **currentView** and **currentIndex** properties of the parent lists. Eg: {{level_1.currentItem.name}}  |
```

Comment:
explain if 1 is outermost or innermost `(from 1 through 3, where 1 is the outermost list)`

Reply:
updated